components/Discussion/DiscussionPreferences.js

```diff
@@ -208,6 +200,9 @@ class DiscussionPreferencesEditor extends PureComponent {
               </A>
             ))}
           </Interaction.P>
+          <Button onClick={this.onSave}>
+            {t('components/DiscussionPreferences/save')}
+          </Button>
```

Comment:
let's move this above the `<Interaction.P>` to ensure that people do not miss it if they have many `credentialSuggestions`